completions/bash/skopeo

```diff
@@ -1,6 +1,7 @@
 #! /bin/bash
+# shellcheck shell=bash disable=SC2207
```

Comment:
I’m not quite happy about this global override… Do I understand correctly that this is all for the
```sh
COMPREPLY=( $( compgen -W …) )
```
instances?

(Note that the example of `compgen` in `(info bash)` changes `IFS` to parse the output correctly before running the command.)

If these are the only causes of the warning, couldn’t they all call a single helper function that either uses one of the supported replacements (to be explicitly and fully safe), or uses only a _local_ override (to continue to rely on the fact that the sets of options don’t contain any other elements that should/could be split)?

Reply:
Yes, it's for `compgen` instances. I changed them to use `read -r` now.

Reply:
(Absolutely non-blocking: This works, but it’s also non-obvious enough that a helper might be easier to maintain (or at least comment on) than copy&pasting it over the place.)